start-server.sh

```diff
@@ -22,14 +22,14 @@ nginx
 
 su ${USER}
 
-export OCT_DJANGO_PORT=4010
-export OCT_BASE_DIR="/plugin_data"
-export TRANSFORMERS_CACHE="/models/huggingface"
-export TRANSFORMERS_OFFLINE="0"
-export EASYOCR_MODULE_PATH="/models/easyocr"
-export TESSERACT_PREFIX="/models/tesseract"
-export TESSERACT_ALLOW_DOWNLOAD="true"
-export PADDLEOCR_PREFIX="/models/paddleocr"
+export OCT_DJANGO_PORT=${OCT_DJANGO_PORT:-4010}
```

Comment:
Changing this would also require switching 

https://github.com/Crivella/ocr_translate/blob/master/nginx.default

to a template, but it does not make a lot of sense as it is only an internal value (any number works, what matters is then the port set by the `listen` part of the nginix configuration, still the user has to specify their container port forwarding and that is where you actually customize the exposed port.